netbox/netbox/models/mixins.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I suspect this line break was meant to go between lines 3 and 4?

Reply:
It was actually because of an import I removed in a subsequent commit and forgot to remove the line break.